database/migration/src/main/resources/changesets/delete_old_call_caching.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <changeSet id="delete_old_call_caching" author="lmcnatt" dbms="mysql,hsqldb,mariadb,postgresql">
+                <sql>
+                    <![CDATA[
+                        CREATE EVENT IF NOT EXISTS delete_old_call_cache_entries
+                            ON SCHEDULE EVERY 1 DAY
+                            DO
+                                DELETE FROM CALL_CACHING_ENTRY WHERE CREATED_AT < (NOW() - INTERVAL 90 DAY);
```

Comment:
I think we probably want this configurable for non-Terra Cromwells... I'd be fine with shipping the sort reversal first and the cleanup later once we find a good way to do that

Reply:
happy to split out those changes to separate concerns! On the topic of non-terra, there's a way to make a changeset run on a precondition (ie I could wire in a flag from the reference.conf file) or we could add this as a completely separate workload outside of cromwell (not sure where it would live? but possibly cleaner), we can discuss in mobbing?